pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -543,23 +546,24 @@
   <!-- Inputs container for instance_combiner_Dc; each input must have exactly one instance child (DC combiner or string) -->
   <xs:complexType name="combiner_dc_inputs_type">
     <xs:sequence>
+      <!-- Do we need this input? -->
       <xs:element name="input" type="combiner_dc_input_type" minOccurs="1" maxOccurs="unbounded"/>
     </xs:sequence>
   </xs:complexType>
   
-  <!-- Input for instance_combiner_dc/inputs elements; it must have exactly one child chosen from the allowed types -->
+  <!-- Input for instance_combiner_dc/inputs elements; it must have one or more strings or one or more combiner boxes -->
   <xs:complexType name="combiner_dc_input_type">
-    <xs:choice minOccurs="1" maxOccurs="unbounded">
-      <xs:element name="instance_combiner_dc" type="instance_combiner_dc_type"/>
-      <xs:element name="string" type="string_type"/>
+    <xs:choice>
+      <xs:element name="instance_combiner_dc" type="instance_combiner_dc_type" maxOccurs="unbounded"/>
+      <xs:element name="string" type="string_type" maxOccurs="unbounded"/>
     </xs:choice>
   </xs:complexType>
 
   <!-- Instance Optimizer: must have exactly one cable element (choosing between cable_to_parent or cable_to_neighbor)
        and exactly one inputs child that contains one or more input children (each wrapping a module_layout) -->
   <xs:complexType name="instance_optimizer_type">
     <xs:sequence>
-      <xs:choice minOccurs="1" maxOccurs="2">
```

Comment:
Again, discrepancy between comment and implementation. I'm guessing we want `maxOccurs="1"`, which is default